services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Extensions.kt

```diff
@@ -9,6 +10,13 @@ fun Any?.toJsonString(): String = OBJECT_MAPPER.writeValueAsString(this)
 fun Any?.toJsonPrettyString(): String =
     OBJECT_MAPPER.writerWithDefaultPrettyPrinter().writeValueAsString(this)
 
+fun String.getResource() =
+    // The class here is arbitrary, could be any class. Using ClassLoader might be cleaner, but it
```

Comment:
I would remove this comment, feel free to ignore.

Reply:
I think this might be useful so that no one wonders what's special about the JsonNode class.